spec/system/task_spec.rb

```diff
@@ -1,24 +1,23 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { FactoryBot.create(:project) }
+  let(:task) { FactoryBot.create(:task, project_id: project.id) }
```

Comment:
Factoryにassociationを定義するとproject_idを指定する必要が無くなります。
```suggestion
  let(:task) { create(:task) }
```

Reply:
コミットID[41e9105](https://github.com/SakitaDaiki0623/rspec_app_exam/pull/1/commits/41e9105e1d1d509ec6f7c7368e49b9bcd2c2f1f6)で修正しました。